softioc/device.py

```diff
@@ -162,9 +184,19 @@ def init_record(self, record):
             recGblResetAlarms(record)
         return self._epics_rc_
 
+    def __completion(self, record):
+        '''Signals that all on_update processing is finished'''
+        if self._blocking:
+            signal_processing_complete(record, self._callback)
+        pass
```

Comment:
Forgotten relic!

Reply:
Fixed in a0bce0ce330bb96274d4bbebff179c872165d7e6